spec/controllers/pageflow/atom_feed_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'spec_helper'
+
+module Pageflow
+  describe EntriesController do
```

Comment:
At the moment files in `spec/controllers` are more or less one-to-one with files in `app/controllers`. I agree, though, that `entries_controller_spec.rb` is already very long. What do you think about putting this into `spec/controllers/pageflow/entries_controller/atom_feed_spec.rb`? That way, it's still clear which controller we are testing.

Reply:
Sure, fine with me.

Actually, the `atom` action has nothing in common with the `html` action. It should really be a separate action or a separate controller, even. Leaving this as-is for now.